bin/setup

```diff
@@ -6,3 +6,4 @@ set -v
 ROOT_DIR=$(cd $(dirname $0)/..; pwd)
 
 bundle install --gemfile=${ROOT_DIR}/Gemfile
+gem install rbs:$(bundle exec gem list rbs | ruby -e 'puts gets[/([0-9.]+)/, 1]')
```

Comment:
We must install `rbs` gem globally to execute RBS without bundler context.
It is necessary for here: https://github.com/ruby/gem_rbs_collection/pull/502/files#diff-4e821476d5e4f5626c6f9fca337a47afe96a5e3daf23bccb02ad0bb109a9a407R106


`rbs collection install` command needs `Gemfile.lock` that includes the test target gems to generate `rbs_collection.lock.yaml`. But the `rbs` command should be executed in the different Bundler context than the `Gemfile.lock`. Because the `Gemfile.lock` for `rbs_collection.lock.yaml` cannot contain `rbs` gem. This `Gemfile.lock` is just generated by `bundle lock` command, so no gems are installed from the `Gemfile.lock`.





I think it is better if we can avoid the global installation. But I have no idea to avoid it 😓

Reply:
You may want to give [`bundler-compose`](https://github.com/rubygems/bundler-compose) a try?

Reply:
We've discussed `bundler-compose` and it is not appropriate for this situation.

* We require gems must not be installed, but `bundler-compose` wants to install gems
* If we use `bundler-compose`, we need to install `bundler-compose` gem globally

Another point, `rbs` gem is always installed globally because it is a bundled gem. So, the global installation does not have a large impact.

BTW, if the user installs Ruby with an OS package manager, `gem install rbs` will require `sudo` permission. This command will not work well in this case, but I think it is a rare case.